opencv-devel was not able to install in multilib environment, because
OpenCVConfig.cmake was different on 32 and 64 bit arch. Moving this file
to libdir will solve this (similar fix is used in Fedora for long time.
Resolves: #658060

diff -up OpenCV-2.0.0/CMakeLists.txt.multilib OpenCV-2.0.0/CMakeLists.txt
--- OpenCV-2.0.0/CMakeLists.txt.multilib	2013-04-05 09:24:01.000000000 +0200
+++ OpenCV-2.0.0/CMakeLists.txt	2013-04-05 09:36:20.892741789 +0200
@@ -34,7 +34,7 @@ set(CMAKE_SHARED_LINKER_FLAGS_RELWITHDEB
 
 project(OpenCV)
 
-cmake_minimum_required(VERSION 2.4)
+cmake_minimum_required(VERSION 2.6.3)
 
 if(MSVC)
     set(CMAKE_USE_RELATIVE_PATHS ON CACHE INTERNAL "" FORCE)
@@ -738,8 +738,13 @@ exec_program(mkdir ARGS "-p \"${CMAKE_BI
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/OpenCVConfig.cmake.in" "${CMAKE_BINARY_DIR}/unix-install/OpenCVConfig.cmake" IMMEDIATE @ONLY)
 
 if(UNIX)
-    # For a command "FIND_PACKAGE(FOO)", CMake will look at the directory /usr/share|lib/FOO/FOOConfig.cmake, so:
-    install(FILES ${CMAKE_BINARY_DIR}/unix-install/OpenCVConfig.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/share/opencv/)
+    #http://www.vtk.org/Wiki/CMake/Tutorials/Packaging reference
+    # For a command "find_package(<name> [major[.minor]] [EXACT] [REQUIRED|QUIET])"
+    # cmake will look in the following dir on unix:
+    #                <prefix>/(share|lib)/cmake/<name>*/                     (U)
+    #                <prefix>/(share|lib)/<name>*/                           (U)
+    #                <prefix>/(share|lib)/<name>*/(cmake|CMake)/             (U)
+    install(FILES ${CMAKE_BINARY_DIR}/unix-install/OpenCVConfig.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/cmake/opencv/)
 endif()
 
 # --------------------------------------------------------------------------------------------
